test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -167,10 +167,10 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 			// lifecycle in windows, to understand it we first analyze what the volumemanager does:
```

Comment:
This comment also needs to be updated, as it's not just a windows problem

Reply:
I updated the comment to read a bit better for all distros.